owasp-suppressions.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+  <suppress>
+    <notes><![CDATA[
+   Any hypertrace dep
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.hypertrace\..*@.*$</packageUrl>
+    <cpe>cpe:/a:grpc:grpc</cpe>
+    <cpe>cpe:/a:utils_project:utils</cpe>
+  </suppress>
+
+  <!-- This is the final version, and its coming via transitive dep. -->
+  <suppress>
+    <notes><![CDATA[
+    We need to get pinot to upgrade this dep, it's 16 years old. Upgrades fix this issue but have changed package/group names
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/commons\-httpclient/commons\-httpclient@.*$</packageUrl>
+    <cve>CVE-2012-5783</cve>
+    <cve>CVE-2020-13956</cve>
+  </suppress>
+
+  <!-- This is the latest version -->
```

Comment:
Latest version isn't a reason to suppress an identifer. If there's a vulnerability in a package that hasn't been fixed in the latest version, we'd suppress that specific vulnerability with an explanation and put a relatively short expiration to revisit.

Reply:
Looks like we got the CPE->CVE switch in, but please also add the expiration and explanation since these are issues that should eventually be fixed.

Reply:
Done